tests/console/validate_fs_table.pm

```diff
@@ -0,0 +1,65 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Validation of filesystem table type, filesystem partitioning, reading and writting in data or root partitions, check of unpartitioned disk space. 
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+
+    select_console "root-console";
```

Comment:
Please check this https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9342/files.

Reply:
I will try to implement it in the test, once it's merged.